ui/lib/kubernetes/addon/routes/configuration.js

```diff
@@ -1,7 +1,17 @@
-import FetchConfigRoute from './fetch-config';
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { withConfig } from '../decorators/fetch-config';
+
+@withConfig(true)
```

Comment:
Why are we passing true here?

Reply:
Ahh good catch! There was a reason early on that I scrapped :smile: